doc/index.rst

```diff
@@ -1,12 +1,136 @@
-ts_ATWhiteLightSource documentation preview
-###########################################
+.. py:currentmodule:: lsst.ts.atwhitelight
 
-.. This page is for local development only. It isn't published to pipelines.lsst.io.
+.. _lsst.ts.atwhitelight:
 
-.. Link the index pages of package and module documentation directions (listed in manifest.yaml).
+####################
+lsst.ts.atwhitelight
+####################
+
+.. image:: https://img.shields.io/badge/SAL\ Interface-gray.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/ATWhiteLight.html
+.. image:: https://img.shields.io/badge/GitHub-gray.svg
+    :target: https://github.com/lsst-ts/ts_atwhitelight
+.. image:: https://img.shields.io/badge/Jira-gray.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_atwhitelight
+
+.. _lsst.ts.atwhitelight.overview:
+
+Overview
+========
+
+ts_atwhitelight controls the Vera C. Rubin Observatory Auxiliary Telescope white light calibration source.
+
+This light source feeds a monochromator (ATMonochromator), which in turns illuminates the flat field screen.
+
+The white light system is comprised of the following components:
+
+* A KiloArc lamp controller that controls a mercury-vapor lamp with a range 800-1200W. We have two manuals for this: a :download:`user guide <KiloArc Lamp with optional computer control manual rev N.pdf>` and a supplement describing the :download:`remote control option <Remote control option for the KiloArc.pdf>`.
+
+* A ThermoTek T257P chiller. We have two manuals for this: A :download:`user guide <T257P User Manual_Rev X1_02.pdf>` and a detailed description of the :download:`serial communication protocol <TTK Serial Communication Protocol.pdf>`.
+
+* A shutter controlled by a linear actuator.
+
+* A `LabJack T4 <https://labjack.com/products/t4>`_ I/O module to control the lamp controller and linear actuator.
+
+.. _lsst.ts.atwhitelight.preserving_bulb_life:
+
+Preserving Bulb Life
+--------------------
+
+The CSC enforces the following rules, in order to extend the life of these expensive bulbs:
+
+* You may not turn on the lamp unless the chiller is running.
+
+* Warmup phase: the lamp must remain on for at least config.lamp.warmup_duration=900 seconds before you can turn it off without forcing it.
+  This gives the mercury time to fully evaporate.
+  If absolutely necessary you can force the lamp off sooner by issuing the turnLampOff command with force=True.
+  The CSC will force the lamp off and go to fault state if it detects a problem with the chiller or lamp controller.
+
+* Cooldown phase: the lamp must remain off for least config.lamp.cooldown_duration=900 seconds before you can turn it on again.
+  This gives the mercury time to fully recondense.
+
+* You may not turn off the chiller while the lamp is on or cooling down.
+
+In addition, the Operating Parameters section of the manual says the following for the He-Xe bulbs:
+
+* Average bulb lifetime is 1500 hours when operated at 1000 W.
```

Comment:
Is there some indication bulb shall be replaced? Or fault state it's not on? If yes, can you please add details?

Reply:
Unfortunately there is no such info that I know of. The lamp controller is very primitive and of the error codes it supports, none seem to indicate that the bulb has burned out.

I wish we had a photodiode or similar that directly detected if the lamp was on. Especially since one of the lamp controller's error codes is LAMP_STUCK_ON (with value 7, so it takes 7 seconds to parse that!).